src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -227,6 +229,27 @@ _nns_edge_cmd_clear (nns_edge_cmd_s * cmd)
   }
 }
 
+/**
+ * @brief Validate edge command.
+ */
```

Comment:
How about set `NNS_EDGE_MAGIC_DEAD` when command is cleared?
`_nns_edge_cmd_clear()`

Reply:
updated